auth_supplier/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 {
     'name': "Auth Supplier",
     'category': 'Portal',
-    'version': '9.0.2.0.0',
+    'version': '10.0.1.0.0',
```

Comment:
Please see https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0#tasks-to-do-in-the-migration with the reference for the change of the version, and there's a rationale after this decision that is too large to explain here.